### PR TITLE
Mechanism to copy text to the player's clipboard

### LINKED ISSUE #1391: [Suggestion] /gt util hand should copy the meta id to clipboard
**Is your feature request related to a problem? Please describe.**
After running the command `/gt util hand`, the metaitem name is output into the chat, after which you still have to type out this metaitem name into a script.

**Describe the solution you'd like**
It would be nice if the usage of the `/gt util hand` command could copy the item's metaname into your clipboard, similar to the `/ct hand` command from Crafttweaker. This would save having to type out the item's metaitem name into the script, and instead simply paste the name into the script

### PR DESCRIPTION
**Pull request by warjort**
**What:**
Add a generic mechanism to copy text to the player's clipboard and use it in the "gt util hand" command for the metavalueitem's name

**How solved:**
Created a new gregtech.api.net.PacketClipboard which is just a string.

Added 2 new util methods
ClipboardUtil.copyToClipboard(String) - copies the string to the clipboard if the AWT desktop is available
ClipboardUtil.copyToClipboard(EntityPlayerMP, String) - sends the network packet to the client which will invoke the other method

When "/gt util hand" sends the chat message to display the MetaValueItem.unlocalizedName it now also sends that value to the player's clipboard

**Outcome:**
Closes #1391

### CODE REVIEW

**Review by LAGIdiot — CHANGES_REQUESTED** [review]
Good work on this PR.

Please move all newly added content from `GTUtility` to `ClipboardUtil` so it is in own file.

**Comment by warjort** [comment]
Done

File: src/main/java/gregtech/api/util/GTUtility.java [hunk 1]
```diff
@@ -50,6 +50,7 @@
 import net.minecraftforge.items.IItemHandlerModifiable;
 
 import javax.annotation.Nullable;
+
```
Comment (LAGIdiot) [inline src/main/java/gregtech/api/util/GTUtility.java:53]: Please don't do unnecessary changes to files which are not directly related to given changes.

I know you were touching this file in previous commits but right now there is no to touch this file now.

**Review by LAGIdiot — CHANGES_REQUESTED** [review]
Thanks for updating this PR. I have find one small thing which needs to be fixed.

**Review by LAGIdiot — APPROVED** [review]
Thank you for finishing this up.

**Comment by LAGIdiot** [comment]
@warjort this should work every time or is there some specific case when this does not work?

**Comment by warjort** [comment]
It only copies to the clipboard when MetaValueItem.unlocalizedName is sent to chat.
Not all blocks do that.

**Comment by LAGIdiot** [comment]
Oh I see, thanks for info.

**Pull request merged**